src/test/java/com/synopsys/integration/detect/battery/tests/CpanmBattery.java

```diff
@@ -0,0 +1,21 @@
+package com.synopsys.integration.detect.battery.tests;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.detect.battery.BatteryTest;
+
+@Tag("battery")
+public class CpanmBattery {
+    @Test
+    void lock() {
+        final BatteryTest test = new BatteryTest("cpanm-lock");
+        test.sourceDirectoryNamed("windows-cpanm");
+        test.sourceFileFromResource("composer.json");
+        test.sourceFileFromResource("composer.lock");
+        test.git("https://github.com/petrkle/zonglovani.info.git", "HEAD");
```

Comment:
Customers have expressed that they do not want their project, even open-sourced ones, used in our tests. Now this repo may not be a customer, but we should still try to scrub the urls if we can. Is this project cloned for the battery test?

Reply:
If it is open source then it shouldn't be an issue, but again we can just make them up if we are told to.